docs-2.0/nebula-explorer/deploy-connect/ex-ug-deploy.md

```diff
@@ -25,7 +25,11 @@
 
    | 安装包 | 检验和 | Nebula Graph内核版本 |
    | ----- | ----- | ----- |
-   | [nebula-graph-explorer-{{ explorer.base100 }}-1.x86_64.rpm](https://oss-cdn.nebula-graph.com.cn/nebula-graph-explorer/nebula-graph-explorer-{{ explorer.base100 }}-1.x86_64.rpm) |  [nebula-graph-explorer-{{ explorer.base100 }}-1.x86_64.rpm.sha256](https://oss-cdn.nebula-graph.com.cn/nebula-graph-explorer/nebula-graph-explorer-{{ explorer.base100 }}-1.x86_64.rpm.sha256) | 2.0.1 |
+   | [nebula-graph-explorer-{{ explorer.base100 }}-1.rpm]() |  [nebula-graph-explorer-{{ explorer.base100 }}-1.rpm.sha256]() | 2.0.1 |
```

Comment:
2.0.1 -> nebula.release

DO NOT hardcode. As you were learned in the colleage.

Reply:
I've got it.